src/app-layout/visual-refresh/trigger-button.tsx

```diff
@@ -1,24 +1,43 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
-import React from 'react';
+import React, { Ref } from 'react';
 import clsx from 'clsx';
 
 import { ButtonProps } from '../../button/interfaces';
+import { InternalButton } from '../../button/internal';
 import { IconProps } from '../../icon/interfaces';
 import Icon from '../../icon/internal';
+import { useAppLayoutInternals } from './context';
 
 import styles from './styles.css.js';
 
 export interface TriggerButtonProps {
   ariaLabel?: string;
   className?: string;
+  /**
+   * The name of the Icon.
+   *
+   * An icon is neccessary for this component to render. If the name is not given there must be a iconSvg prop passed or the component will not render
+   */
   iconName?: IconProps.Name;
+  /**
+   * A svg node for a custom icon.
+   *
+   * An icon is neccessary for this component to render. If the SVG is not given there must be a iconName prop passed or the component will not render
+   */
   iconSvg?: React.ReactNode;
```

Comment:
What is the fallback for the case an icon is missing? Should we add a warning?

Reply:
Good question.  I am removing those comments. The button will render with no icon if none provided.

Reply:
removed docstrings for now